fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoTab.java

```diff
@@ -0,0 +1,46 @@
+package com.github.dedis.popstellar.ui.detail;
+
+import androidx.annotation.IdRes;
+
+import com.github.dedis.popstellar.R;
+
+import java.util.Arrays;
+import java.util.List;
+
+/** Enum where each element represent a tab in LaoActivity */
+public enum LaoTab {
+  EVENTS(R.id.lao_detail_event_list_menu),
```

Comment:
maybe worth having a 'default' tab, that is just an alias of one of these tabs ?

Reply:
I am not certain I understand what you mean by that. Could you be more precise ?